js/ddmlib/property_formatter.js

```diff
@@ -85,6 +85,9 @@ const formatProperties = function(root /* ViewNode */) {
         if (node.name == undefined) {
             node.name = node.classname
         }
+        if (node.name == undefined && classNames) {
```

Comment:
Does the code above work, or does node.name never equal undefined by the time it gets there? Should it be the following instead?
`
        if (node.name == undefined) {
            node.name = (classNames) ? classNames[node.classnameIndex] + "@" + node.hashcode : node.classname;
        }
`

Reply:
in the new proto format, `node.classname` is no longer defined. So `node.name` stays undefined in that case